web/front/src/styles/_responsiveHome.sass

```diff
@@ -0,0 +1,68 @@
+@use '_var.sass' as var
+
+
+
+@media (max-width: 480px)
+  .App
+    padding: var.$gap-s
+    .navbar
+        margin-bottom: var.$gap-l
+        .logo
+            display: flex
+            align-items: center
+            .nav-logo 
+                width: 50px
+                margin-right: 5px
+            .site-title
+                font-size: var.$button-text-size
+                vertical-align: baseline
+                margin-right: var.$gap
+
+    .button-primary 
+        font-size: .75rem
+        padding: 15px var.$gap-xs - 10px
+
+
+    .page
+        .header
+            margin-bottom: var.$gap-l
+            h1
+                font-size: var.$subtitle-size
```

Comment:
The variable $h1-size is in rem value so it should be responsive by default ? Did you have a problem with it ?